source/docs/software/advanced-control/state-space/state-space-flywheel-walkthrough.rst

```diff
@@ -1,4 +1,123 @@
-State-Space Controller Examples
-===============================
+State-Space Controller Walkthrough
+==================================
 
+.. note:: Before following this tutorial, readers are recommended to have read an :ref:`docs/software/advanced-control/state-space/state-space-intro:Introduction to state-space control`.
 
+The goal of this tutorial is to provide "end-to-end" instruction on implementing a state-space controller on a flywheel.  By following this tutorial, readers will learn how to:
+
+1. Create an accurate state-space model of a flywheel using :term:`system identification` or CAD software
+2. Implement a Kalman Filter to laglessly filter encoder velocity measurements.
+3. Implement a :ref:`LQR <docs/software/advanced-control/state-space/state-space-intro:The Linear-Quadratic Regulator>` feedback controller which, when combined with model-based feedforward, will generate voltrage :term:input:s to drive the flywheel to a :term:`reference`.
+
+This tutorial is intended to be approachable for teams without a great deal of programming expertise.  While the WPILib library offers significant flexibility in the manner in which its state-space control features are implemented, closely following the implementation outlined in this tutorial should provide teams with a basic structure which can be reused for a variety of state-space systems.
+
+.. todo:: link to allwpilib example
+
+Why use state-space control?
+============================
+
+Because IIC is a meme award. Thank you for coming to my ted talk
+
+.. todo:: real explication.
+
+Modeling our flywheel
+=====================
+
+The idea of modeling the :term:`dynamics` of physical systems using a system of equations is central to state-space control and modern control theory. Recall that continuous state-space systems are modeled using the following system of equations:
```

Comment:
Add an embedded hyperlink to "Recall" with a link to the introduction to state-space article.